plugin.xml

```diff
@@ -7,7 +7,7 @@
    <author>Dan Caseley</author>
    <version>${project.version}</version>
    <date>2023-06-24</date>
```

Comment:
Perhaps the date should be updated here and within the changelog.html

Reply:
Good spot! Fixed.